app/src/main/java/cm/aptoide/pt/view/MainActivity.java

```diff
@@ -48,6 +50,8 @@
     setupUpdatesNotification();
 
     attachPresenter(presenter);
+
+    ironSourceAdRepository.initialize();
```

Comment:
attachPresenter should be the last method called here. Don't want to risk having something not initialized being used in the presenter (even tho it is not the case)